types/react-bootstrap-typeahead/index.d.ts

```diff
@@ -424,7 +424,7 @@ export const Loader: React.FunctionComponent<LoaderProps>;
 --------------------------------------------------------------------------- */
 export interface AutosizeInputProps extends Pick<React.InputHTMLAttributes<HTMLInputElement>, "className" | "style"> {
     inputClassName?: string | undefined;
-    inputRef?: React.LegacyRef<HTMLInputElement> | undefined;
+    inputRef?: React.Ref<HTMLInputElement> | undefined;
```

Comment:
This was most likely a bug. Having a string ref here would mean the ref is assigned to `AutosizeInput` iff it is a class component not to the owner that's rendering `AutosizeInput`